common.lic

```diff
@@ -268,9 +268,10 @@ module DRC
     end
   end
 
-  def hide?
+  def hide?(*hide_type)
+    hide_type.any? ? hide_type = hide_type[0] : hide_type = 'hide' 
```

Comment:
Although this is drifting farther from the purpose of base-empty perhaps, maybe we should just let them error out and fix it. @rpherbig opinions?